docs/advanced-topics/negative-mining.md

```diff
@@ -0,0 +1,86 @@
+(negative-mining)=
+# {octicon}`telescope` Negative Mining
+
+## Context: Deep Metric Learning
+
+Metric Learning algorithms attempt to teach the Deep Neural Network (DNN) to tell
+which objects are semantically/visually similar and which ones are not.
+
+For uni-modal fine-tuning tasks such as text-to-text, image-to-image, or mesh-to-mesh,
```

Comment:
```suggestion
For single-mode fine-tuning tasks such as text-to-text, image-to-image, or mesh-to-mesh,
```

Reply:
what's wrong with unimodal? i think it's more common than single-model